scripts/update_configs.py

```diff
@@ -20,25 +20,44 @@
 
     python -m scripts.update_configs
 """
+
 from __future__ import absolute_import  # pylint: disable=import-only-modules
 from __future__ import unicode_literals  # pylint: disable=import-only-modules
 
+import getpass
 import os
 import re
+import sys
 
 import python_utils
+import release_constants
+
+import requests
```

Comment:
Is this an inbuild lib import, if so should we move this above?

Reply:
I placed it above but I get a lint error with incorrect sorting of imports.
/cc @oppia/dev-workflow-team